packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -332,7 +337,13 @@ def setup_vip_workload_tests(dcos_api_session, container, vip_net, proxy_net, ip
     return tests
 
 
+def create_app_prefix(vip_net, proxy_net, container, ipv6, named_vip):
```

Comment:
After your last change, the function uses only 2 out of 5 arguments. I'd recommend to merge it into `vip_workload_test` as a one-liner.